src/plugin.ts

```diff
@@ -20,10 +18,10 @@ export async function plugin(inputs: PluginInputs, env: Env) {
     octokit,
     env,
     logger: new Logs("debug"),
-    adapters: {} as ReturnType<typeof createAdapters>,
   };
 
-  context.adapters = createAdapters(dataSource, context);
-  await updatePullRequests(context);
-  return proxyCallbacks[inputs.eventName](context, env);
+  context.logger.info(
+    `Will check the following repos / orgs: [${context.config.repos.monitor.join(", ")}] and ignore [${context.config.repos.ignore.join(", ")}]`
+  );
+  return await updatePullRequests(context);
```

Comment:
Without any checks this runs on any push event such as new branch creation, is this intended yeah?

Reply:
This is supposedly only called by the events defined within the manifest. Since `runsOn` can override them, there is no way to know in advance what kind of events will trigger the plugin. And for this one it is not really relevant to check I think, since we gonna need to tune the amount of time this runs.